### PR TITLE
Example with Solidity

### PR DESCRIPTION
**Pull request by hswick**
Added full example with smart contract, example queries contract and changes state of contract, documented solidity compiling on README

### CODE REVIEW

File: examples/simple_storage.rs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+  //Get current balance
+  let balance = web3.eth().balance(accounts[0], None).wait().unwrap();
+
+  println!("Balance: {}", balance);
+
+  // Get the contract bytecode for instance from Solidity compiler
+  let bytecode: Vec<u8> = include_str!("/build/SimpleStorage.bin").from_hex().unwrap();
+  // Deploying a contract
+  let contract = Contract::deploy(web3.eth(), include_bytes!("/build/SimpleStorage.abi")).unwrap()
```
Comment (tomusdrw) [inline examples/simple_storage.rs]: Could you change the paths to be relative?

**Comment by tomusdrw** [comment]
Looks good, thank you for the contribution. Could you update the paths to be relative? This should make CI happy :)

**Comment by hswick** [comment]
@tomusdrw could you specify what you mean by relative? I thought they already were.

**Comment by tomusdrw** [comment]
@hswick A dot is missing before first leading slash
```
include_bytes!("/build/SimpleStorage.abi")
// should be
include_bytes!("./build/SimpleStorage.abi")
```

You can see that tests fail because the path cannot be found (it's looking for relative path of `/build/SimplStorage.abi`

**Comment by hswick** [comment]
Ah yes I see, wasn't sure if it was that obvious or not

**Comment by tomusdrw** [comment]
Thanks!

**Pull request merged**